xwiki-platform-core/xwiki-platform-bootstrap/src/main/less/navs.less

```diff
@@ -77,14 +76,16 @@
 // Give the tabs something to sit on
 .nav-tabs {
   border-bottom: 1px solid @nav-tabs-border-color;
+  display: flex;
+  gap: .2em;
```

Comment:
You turned `2px` into `.2em`? Or how did you arrive at that number? Wouldn't it make sense to have some standard/variable for this kind of distance to make sure we have some uniformity?